src/patternfly/components/VerticalNav/styles.scss

```diff
@@ -0,0 +1,127 @@
+@import "../../patternfly-utilities";
+
+.pf-c-vertical-nav {
+  /* stylelint-disable */
+
+  // Vertical nav
+  --pf-c-vertical-nav--Color:             var(--pf-global--Color--dark-100);
+  --pf-c-vertical-nav--BackgroundColor:   var(--pf-global--BackgroundColor--light-100);
+
+  // Vertical nav link
+  --pf-c-vertical-nav__link--PaddingRight:    var(--pf-global--spacer--xl);
+  --pf-c-vertical-nav__link--PaddingLeft:     var(--pf-global--spacer--xl);
+  
+  // Vertical nav link active :after
+  --pf-c-vertical-nav__link--after--active--Width:   2rem;
+  --pf-c-vertical-nav__link--after--active--Height:  .25rem;
+  --pf-c-vertical-nav__link--after--active--Color:   var(--pf-global--Color--active);
+
+  // Vertical nav link colors
+  --pf-c-vertical-nav__link--Color:             var(--pf-global--Color--dark-100);
+  --pf-c-vertical-nav__link--hover--Color:      var(--pf-global--Color--dark-100);
+  --pf-c-vertical-nav__link--focus--Color:      var(--pf-global--Color--dark-100);
+  --pf-c-vertical-nav__link--active--Color:     var(--pf-global--Color--active);
+  --pf-c-vertical-nav__link--disabled--Color:   var(--pf-global--Color--disabled);
+
+  // Vertical nav link background colors
+  --pf-c-vertical-nav__link--BackgroundColor:           var(--pf-global--BackgroundColor--light-100);
+  --pf-c-vertical-nav__link--hover--BackgroundColor:    var(--pf-global--BackgroundColor--light-100);
+  --pf-c-vertical-nav__link--focus--BackgroundColor:    var(--pf-global--BackgroundColor--light-100);
+  --pf-c-vertical-nav__link--active--BackgroundColor:   var(--pf-global--BackgroundColor--light-100);
+  --pf-c-vertical-nav__link--disabled--BackgroundColor: var(--pf-global--BackgroundColor--disabled);
+
+  // Vertical nav link text padding
+  --pf-c-vertical-nav__link__text--PaddingTop:     var(--pf-global--spacer--lg);
+  --pf-c-vertical-nav__link__text--PaddingBottom:   var(--pf-global--spacer--lg);
+
+  // Vertical nav link arrow
+  --pf-c-vertical-nav__link__arrow--Top: calc(var(--pf-c-vertical-nav__link--PaddingTop) - .4rem);
+  --pf-c-vertical-nav__link__arrow--Right: var(--pf-global--spacer--xl);
+
+  /* stylelint-enable */
+
+  // List
+  color: var(--pf-c-vertical-nav--Color);
+  background-color: var(--pf-c-vertical-nav--BackgroundColor);
+
+  // Link
+  &__link {
+    // Align any additional items included in __link
+    display: flex;
+    align-items: baseline;
+    padding-right: var(--pf-c-vertical-nav__link--PaddingRight);
+    padding-left: var(--pf-c-vertical-nav__link--PaddingLeft);
+    color: var(--pf-c-vertical-nav__link--Color);
+    text-decoration: none;
+    background-color: var(--pf-c-vertical-nav__link--BackgroundColor);
+
+    // // States
+    &:hover,
+    &:active,
+    &:focus {
+      text-decoration: none;
+    }
+
+    // Using this method to align active marker to text rather than link. Accomodates <a>'s without icons
+    &__text {
```

Comment:
should this be `pf-c-vertical-nav__text` or `pf-c-vertical-nav__link-text` instead of `pf-c-vertical-nav__link__text` ?

Reply:
`pf-c-vertical-nav__link-text` Yes, it should